jenkins/jobs/dev_env_integration_tests.pipeline

```diff
@@ -27,10 +27,11 @@ script {
 pipeline {
   agent { label agent_label }
   environment {
-    METAL3_CI_USER="metal3ci"
```

Comment:
we dont use this USER anymore ?

Reply:
Nope we do not this variable anymore. metal3ci user is still used in tests but we do not need to specify it via dynamic workflow.